portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,11 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
     <title>MM Portfolio</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBjTF_cxIVfkP_Z9NeXTcUJFNHiLjeYDhk"></script>
```

Comment:
uber nit that i didn't catch before - might make sense to order `<script>` elements together and alphabetically, but we can follow up about that. no need to do anything now.

Reply:
sounds good! will do!